### PR TITLE
Add basic functionality for pointinterval plots from hubverse quantile tables

### PR DESCRIPTION
**Pull request by dylanhmorris**

### CODE REVIEW

File: R/pivot_hubverse_quantiles_wider.R [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      "The following requested pivot quantiles ",
+      "could not be found: {missing_pivot_quantiles}."
+    ))
+  }
+
+  if (is.null(names(pivot_quantiles))) {
+    names(pivot_quantiles) <- paste("q", pivot_quantiles, sep = "")
+  }
+
+  pivot_quant_map <- setNames(names(pivot_quantiles), pivot_quantiles)
```
Comment (damonbayer) [inline R/pivot_hubverse_quantiles_wider.R:54]: Does this do anything?

File: R/pivot_hubverse_quantiles_wider.R [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    ))
+  }
+
+  dat <- hubverse_table |>
+    dplyr::filter(.data$output_type == "quantile") |>
+    dplyr::mutate("output_type_id" = as.numeric(.data$output_type_id))
+
+  pivot_quantiles_present <- pivot_quantiles %in% hubverse_table$output_type_id
+
+  if (!all(pivot_quantiles_present)) {
```
Comment (damonbayer) [inline R/pivot_hubverse_quantiles_wider.R:40]: I think this is better done with `setdiff`

File: R/pivot_hubverse_quantiles_wider.R [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  if (is.null(names(pivot_quantiles))) {
+    names(pivot_quantiles) <- paste("q", pivot_quantiles, sep = "")
+  }
+
+  pivot_quant_map <- setNames(names(pivot_quantiles), pivot_quantiles)
+
+  dat <- dat |>
+    dplyr::filter(.data$output_type_id %in% !!pivot_quantiles) |>
+    dplyr::mutate(
+      "which_quantile" = dplyr::recode(
```
Comment (damonbayer) [inline R/pivot_hubverse_quantiles_wider.R:59]: Recode is superseded by case_match. https://dplyr.tidyverse.org/reference/case_match.html

Although I think you can just do this with indexing?

File: R/pivot_hubverse_quantiles_wider.R (hunk 1 above)
Comment (dylanhmorris) [reply R/pivot_hubverse_quantiles_wider.R:54]: this is the inverse mapping, for supplying to `recode` with `!!!`

File: R/pivot_hubverse_quantiles_wider.R (hunk 2 above)
Comment (dylanhmorris) [reply R/pivot_hubverse_quantiles_wider.R:40]: Good point.

File: R/pivot_hubverse_quantiles_wider.R (hunk 3 above)
Comment (dylanhmorris) [reply R/pivot_hubverse_quantiles_wider.R:59]: created #17

**Pull request merged**